src/macaron/__main__.py

```diff
@@ -132,10 +133,31 @@ def perform_action(action_args: argparse.Namespace) -> None:
             sys.exit(os.EX_USAGE)
 
 
-def main() -> None:
-    """Execute Macaron as a standalone command-line tool."""
+def main(argv: list[str] | None = None) -> None:
+    """Execute Macaron as a standalone command-line tool.
+
+    Parameters
+    ----------
+    argv: list[str] | None
+        Command-line arguments.
+        If ``argv`` is ``None``, argparse automatically looks at ``sys.argv``.
+        Hence, we set ``argv = None`` by default.
+    """
     main_parser = argparse.ArgumentParser(prog="macaron")
-    main_parser.add_argument("-v", "--verbose", help="Run Macaron with more debug logs", action="store_true")
+
+    main_parser.add_argument(
+        "-V",
+        "--version",
+        action="version",
+        version=f"%(prog)s {importlib_metadata.version('macaron')}",
```

Comment:
Looks like by default `argparse` adds `show program's version number and exit` as the `help` attribute for `--version`. Can you please override that and capitalize the sentence to be consistent with other command-line flags?

Reply:
Fixed in 906276a.